docs/changelogs/CHANGELOG-2.0.md

```diff
@@ -32,9 +33,10 @@
 
 ### Deprecated
 
-
 ### Breaking changes
 
 - Upgrade of Terraform components in issue [#2825](https://github.com/epiphany-platform/epiphany/issues/2825) will make running re-apply with infrastructure break on existing 1.x clusters. The advice is to deploy a new cluster and migrate data. If needed a manual upgrade path is described [here.](../home/howto/UPGRADE.md#terraform-upgrade-from-epiphany-1.x-to-2.x)
+- Kubernetes container runtime changed. Dockershim and Docker are no longer on Kubernetes hosts.
+- Filebeat docker input replaced by container input with new fields.
```

Comment:
Please add short info which fields have been removed/renamed.

Reply:
I will add here only added fields because for add_kubernetes_metadata processor they are still valid: https://www.elastic.co/guide/en/beats/filebeat/current/exported-fields-kubernetes-processor.html

Reply:
If there is no removed/renamed fields, then I would not add this info under "Breaking changes" - unless something is not backward compatible.